ui/app/routes/vault/cluster/metrics.js

```diff
@@ -1,14 +1,52 @@
 import Route from '@ember/routing/route';
 import ClusterRoute from 'vault/mixins/cluster-route';
 import { hash } from 'rsvp';
+import { endOfMonth } from 'date-fns';
+import { parseDateString } from 'vault/helpers/parse-date-string';
+
+const getActivityParams = ({ start, end }) => {
+  // Expects MM-YYYY format
+  // TODO: minStart, maxEnd
+  let params = {};
+  if (start) {
+    let startDate = parseDateString(start);
+    if (startDate) {
+      // TODO: Replace with formatRFC3339 when date-fns is updated
+      params.start_time = Math.round(startDate.getTime() / 1000);
+    }
+  }
+  if (end) {
+    let endDate = parseDateString(end);
+    if (endDate) {
+      // TODO: Replace with formatRFC3339 when date-fns is updated
```

Comment:
if you don't already have a JIRA ticket for this might be a good idea to create one so we don't forget.  And have it dependent on any relevant API ticket from the backend.

Reply:
Good call, done!